capitalize_decorator.rb

```diff
@@ -0,0 +1,8 @@
+# FrozenStringLiteral: true
+require_relative 'base_decorator'
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    super.capitalize
+  end
+end
```

Comment:
- [ ] Great job so far. However, I got this error message when I tried to run your app with some data. I created the missing method (`correct_name`) in the `Person` class to test your app with below code :point_down: 

```
   person = Person.new(22, 'maximilianus')
   person.correct_name
   capitalizedPerson = CapitalizeDecorator.new(person)
   capitalizedPerson.correct_name
   capitalizedTrimmedPerson = 
   TrimmerDecorator.new(capitalizedPerson)
   capitalizedTrimmedPerson.correct_name
```
But below is the error message I got. Kindly look into it and fix.

### Error
![run-error](https://github.com/berylcl/OOP-school-library/assets/26597432/b85a3625-2cdc-48df-bc3c-610921889224)